src/ripple/app/ledger/impl/LedgerToJson.cpp

```diff
@@ -65,6 +65,10 @@ void fillJson(Object& json, LedgerInfo const& info, bool bFull)
     json[jss::hash] = to_string (info.hash);
     json[jss::totalCoins] = to_string (info.drops);
     json[jss::accepted] = ! info.open;
+    json[jss::close_flags] = info.closeFlags;
+
+    if (info.parentCloseTime)
```

Comment:
"agree to disagree" is a general property of Ripple's consensus that allows it to converge quickly, since transactions are not 'precious' (a transaction that doesn't make it into a round will certainly make it into the next round). Coming to a consensus on an integer would take a lot longer than coming to a consensus on a bool. Since the close time is an integer, it is treated specially in order that consensus can be timely. First, we have the close resolution which loosens the requirement that all servers take the exact same integer position. And then, if there is not sufficient agreement on close time within a certain amount of time, we just leave it as "no conclusion."

At least, thats how I *think* it all works.